pkg/cmd/ssh/options.go

```diff
@@ -757,17 +757,20 @@ func waitForBastion(ctx context.Context, o *SSHOptions, gardenClient client.Clie
 			return false, err
 		}
 
-		cond := corev1alpha1helper.GetCondition(bastion.Status.Conditions, operationsv1alpha1.BastionReady)
-
-		if cond == nil || cond.Status != gardencorev1alpha1.ConditionTrue {
-			lastCheckErr = errors.New("bastion does not have BastionReady=true condition")
+		switch cond := corev1alpha1helper.GetCondition(bastion.Status.Conditions, operationsv1alpha1.BastionReady); {
+		case cond == nil:
+			return false, nil
+		case cond.Status != gardencorev1alpha1.ConditionTrue:
+			lastCheckErr = errors.New(cond.Message)
 			fmt.Fprintf(o.IOStreams.ErrOut, "Still waiting: %v\n", lastCheckErr)
 			return false, nil
+		case cond.Status == gardencorev1alpha1.ConditionTrue:
+			// leave switch
```

Comment:
```suggestion
```
now that we do not have the default case anymore, we can also get rid of this case, right?

Reply:
I'm fine to remove it.  it was just to keep code logic reading easily. 
now code will escape `case default` unknown condition and `case cond.Status == gardencorev1alpha1.ConditionTrue:` .  after testing from my local, no any issue at the moment. 8efdc803273ceadc4c4303b280cde5b04763f7bf